panels/user-accounts/um-account-dialog.c

```diff
@@ -1678,3 +1683,21 @@ um_account_dialog_finish (UmAccountDialog     *self,
         g_clear_object (&self->async);
         return user;
 }
+
+void
+um_account_dialog_set_is_small_screen (UmAccountDialog *self,
+                                       gboolean         is_small_screen)
+{
+        if (is_small_screen) {
```

Comment:
Don't think so. We usually call this function only once, since the dialogs are usually recreated when needed.